lib/text/vtt_text_parser.js

```diff
@@ -219,6 +221,46 @@ shaka.text.VttTextParser = class {
     return cue;
   }
 
+  /**
+   * Parses a WebVTT styles from the given payload.
+   *
+   * @param {!shaka.text.Cue} cue
+   * @param {string} payload
+   */
+  static parseCueStyles(cue, payload) {
```

Comment:
I would suggest reversing the arguments since the "cue" param is more of an out parameter.  I would also mention that the given "cue" object will be populated with the settings.

Reply:
I'll change it!